src/background/extension.ts

```diff
@@ -61,7 +61,8 @@ export class Extension {
                 }
                 return this.getConnectionMessage(url, frameURL);
             },
-            onColorSchemeChange: this.onColorSchemeChange,
+            getTabMessage: this.getTabMessage,
+            onColorSchemeChange: this.onColorSchemeChange
```

Comment:
It's OK to leave a comma, it's often used to not highlight unnecessary changes in search results.